data/profiles/artemis.profile

```diff
@@ -0,0 +1,137 @@
+{
+  "actions": [
+    {
+      "documentation": "Toggle trails on or off for satellites around Earth",
+      "gui_path": "/Solar System/Earth",
+      "identifier": "profile.toggle.satellite",
+      "is_local": false,
+      "name": "Toggle satellite trails",
+      "script": "local list = openspace.getProperty('{earth_satellites}.Renderable.Enabled'); for _,v in pairs(list) do openspace.setPropertyValueSingle(v, not openspace.getPropertyValue(v)) end"
+    },
+    {
+      "documentation": "Refocuses the camera on the ISS",
+      "gui_path": "/Solar System/Earth",
+      "identifier": "profile.focus.iss",
+      "is_local": false,
+      "name": "Focus ISS",
+      "script": "openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.Aim', '');openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.Anchor', 'ISS');openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.RetargetAnchor', nil);"
+    },
+    {
+      "documentation": "Retargets the camera on Earth",
+      "gui_path": "/Solar System/Earth",
+      "identifier": "profile.focus.earth",
+      "is_local": false,
+      "name": "Focus on Earth",
+      "script": "openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.Aim', '');openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.Anchor', 'Earth')openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.RetargetAnchor', nil);"
+    }
```

Comment:
Maybe we don't need these actions in the profile?  I think we can get rid of them from the default one as well, tbh